engineering/methodology.md

```diff
@@ -127,73 +95,19 @@ All standups are happening online and asynchronously, per-team in appropriate
 * Fixed max time **per-team: 15min (including questions)**. Agree between teams in advance, if more time needed.
 
-## Keeping track of things
-
-Github Projects board will used as a Kanban board.
-
-Teams managing issues across Github Org borders (Applications, Language Analysis) will use the “add card”  trick, described in dear-github/dear-github#209 to  add issues from different org  to the team’s board.
-
-Issues repositories:
-
-* [src-d/issues-general](https://github.com/src-d/issues-general) - should be used more, as place for feedback/requests (i.e website, etc)
-* [src-d/issues-infrastructure](https://github.com/src-d/issues-infrastructure) - issues/requests to Infrastructure team must be created here.
-
-# Definitions
-
-Below, we'll define the meaning of each Kanban column.
-
-This is not a mere formality - it’s important to have same things in mind when
-moving work from one state to another and what is documented below is a
-consensus, that took a few hours to build among Engineering leads.
-
-## What does TODO mean?
-
-Something an engineer can immediately take and act up on. If the task is not
-possible to start until another (well-known) task is finished first, this task
-shall not be in TODO yet.
-
-*Rationale* there are 2 goals we want to keep in mind, while working with Kanban
-board:
-
-* as small TODO list on a Kanban as possible and
-* tasks, moving fast from end-to-end of Kanban board (short Cycle Time)
-
-This definition and goals also imply that leads should grooming backlog on a
-daily basis.
-
-> Alternative definition (discussed but discarded): all things that are planned to be done ASAP. This would imply that issues can be added to TODO in bulk at once, even in cases when multiple issues of single project have known dependency on one another. Was discarded on the premise that it would go against the 2 goals, listed above.
-
-## What does Done mean?
-
-General mindset: Done always means that a user, both inside and outside src-d,
-is able to use it through well-established interfaces. Detailed definition may
-vary per-team.
-
-Few concrete examples, in case of work done on:
-
-* Golang library - it should be possible to use them though the particular version published on gopkg.in
-* Scala library - it should be possible to use from Maven/sbt build
-* Web site/application - should be accessible through a known (staging/production) environment URL
-* etc
-
-
-## What does Blocker mean?
-
-*General mindset*: blocker means something **unexpected has happened**
-
-Obvious/expected dependencies between sequence of tasks don’t need to be marked
-as blocked.
+## Retrospective Meeting
 
-Issues from backlog that can not be done, due to **a known dependency** should
-**never** go into the TODO column (thus avoid being blocked). I am not able to
-work on it anymore, because it needs something done to move forward.
+Every month this methodology is reviewed in a meeting called Retrospective Meeting
```

Comment:
I think we need to clarify a few things:

- Who attends the engineering retrospective? (CTO, HoA, Team Leads or anyone delegated by Team Leads?)
- Are there per-team retrospectives too? When? Is it fixed schedule or Team Leads schedule them whenever they consider it?